addons/binding/pom.xml

```diff
@@ -67,6 +67,7 @@
     <module>org.openhab.binding.squeezebox</module>
     <module>org.openhab.binding.systeminfo</module>
     <module>org.openhab.binding.systeminfo.test</module>
+    <module>org.openhab.binding.tankerkoenig</module>
```

Comment:
No there is a file called `feature.xml` you should also add your binding
features/openhab-addons/src/main/feature/feature.xml